src/hooks/useLoginForm.ts

```diff
@@ -0,0 +1,67 @@
+import { useState } from 'react';
+
+const EMAIL_REQUIRED_ERROR = 'ID를 입력해주세요.';
+const EMAIL_INVALID_FORMAT_ERROR = 'ID는 이메일 형식으로 입력해주세요.';
+
+const PASSWORD_REQUIRED_ERROR = 'PW를 입력해주세요.';
+const PASSWORD_MIN_LENGTH_ERROR = 'PW는 최소 8글자 이상이어야 합니다.';
```

Comment:
여기서 선언한 상수를 활용해 볼 수 있겠네요 👍

```suggestion
const PASSWORD_MIN_LENGTH_ERROR = `PW는 최소 ${PASSWORD_MIN_LENGTH}글자 이상이어야 합니다.`;
```